DTOParamConverter.php

```diff
@@ -53,11 +47,15 @@ public function apply(Request $request, ParamConverter $configuration)
     {
         try {
             $class = $configuration->getClass();
-            $instance = new $class;
+            $instance = new $class();
             $request->attributes->set($configuration->getName(), $instance);
 
-            foreach ($this->getProperties($instance) as $parameter => $parameterOptions) {
-                $value = $this->getValue($request,
+            $reflectionClass = new ReflectionClass($instance);
+            $this->callEvent($reflectionClass, $instance, PreSet::class);
```

Comment:
Buralara açıklama satırları eklenebilir.

Reply:
Aayet açık abi yapılan işlem aslında, readme de de nasıl kullanıldığı vs belirtilmiş. Nasıl bir yorum satırı yapılabilir eklenebilir abi sence?

Reply:
Evet bizim için gayet açık, ancak burada bir feature var bunu özellikle de ilk defa bu pakete geliştirme yapacaklar (contributors) için 1-2 satır process'i açıklayan cümle yazılması faydalı olur, readme dosyası implementation ile alakalı, burası development. Code bloklarına yorum satırları eklemekten kaçınmayalım bugün gereksiz gibi gelebilir. İlerleyen zamanlarda bunların faydasını görebiliriz. Şöyle bir açıklama olabilir; "PreSet annotation tanımlanan method(lar) otomatik set edilmeden önce burada triger edilir." aklıma gelen ilk cümleyi yazdım. Burada amaç şu ben burada bir değişiklik yapacağım zaman buradaki kod bloğunun tam olarak ne yaptığını bilmem gerekir, aksi takdirde bug oluşturabilirim.